src/Simulation/Type1Core/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// Allow the simulator assembly to use our internal methods
+[assembly: InternalsVisibleTo("Microsoft.Quantum.Simulators.Type1" + SigningConstants.PUBLIC_KEY)]
```

Comment:
Why do the internals need to be visible to the simulator again? Because it needs to implement things that are intrinsic in Q#?

Reply:
Yeah, hopefully this can be removed before the PR merges, per our design idea of having one extra layer of indirection.

Reply:
See the comment below, but the short version is that the work to refactor simulator to target package relationship is now captured by #432, at which point we should be able to remove this AssemblyInfo entirely.